packages/fx-core/src/core/middleware/projectMigrator.ts

```diff
@@ -678,6 +689,24 @@ async function backup(projectPath: string): Promise<void> {
   }
 }
 
+// append folder path to .gitignore under the project root.
+async function addPathToGitignore(projectPath: string, ignoredPath: string) {
+  const relativePath = path.relative(projectPath, ignoredPath);
```

Comment:
`path.relative` uses platform dependent path separator.

on Windows:
![image](https://user-images.githubusercontent.com/9698542/141728015-64ef580b-be85-416a-8f55-6022fbbc2076.png)

But [gitignore](https://git-scm.com/docs/gitignore#_pattern_format) only uses slash (not backslash) for path separator. So this may not work on Windows.

Reply:
Good point, I have only tested in WSL. Let me have a check.